R/adult.R

```diff
@@ -0,0 +1,45 @@
+#' @title Adult Dataset
+#'
+#' @name adult
+#' @aliases adult_test
+#' @aliases adult_train
+#'
+#' @description
+#' Dataset used to predict whether income exceeds $50K/yr based on census data. Also known as "Census Income" dataset
+#' Train dataset contains 13 features and 30178 observations.
+#' Test dataset contains 13 features and 15315 observations.
+#' Target column is "Target".
+#'
+#' @section Pre-processing:
```

Comment:
Is there a reference as to where this kind of preprocessing was used? If yes, add; otherwise ignore this comment

Reply:
I did not find a reference, but I followed some principles from examples in mlr3data like
https://github.com/mlr-org/mlr3data/blob/0b1f8fa1b6775cb39ac67d099550ea4420b3d0af/R/optdigits.R